kite-web-element/src/router.js

```diff
@@ -23,6 +25,8 @@ const router = new Router({
             next({
```

Comment:
The `if` statement one line above is in-secure, i think. Instead of 
```js
if(store.state.auth.user.scope.length > 0) {
  next({
    path: '/containers'
  })
}
```
The line should check for the actual scope, not just any
```js
if(store.state.auth.user.scope.includes("containers")) {
  next({
    path: '/containers'
  })
}
```

Reply:
This is something that I have thought about, and it has had a task in Asana since last week - https://app.asana.com/0/967075527666002/1108730280290298. I plan on working on this at some point either this weekend or early next week. 

However:
Our current implementation of scope doesn't really work well with this idea because we only have 3 scopes. An example of why this wouldn't be great is this - a user with scope 'containers' logs in. They try to navigate to the 'Help' page. A full-fledged system would check if scope includes 'help', not 'containers', but we don't have that. I don't want to check if scope contains 'containers' when they are going to pages that are not the containers page...

A way I have thought to remedy this is - On the front-end when creating users give the admin and 3 options, [user, admin, createAdmin]. If they pick the user option, it creates a scope array with all the scopes needed, maybe ['containers', 'help', 'settings']. This is better than our current system where the admin currently needs to select each scope setting by hand.